code/datums/diseases/pierrot_throat.dm

```diff
@@ -5,62 +5,56 @@
 	cure_text = "Banana products, especially banana bread."
 	cures = list("banana")
 	cure_chance = 75
-	agent = "H0NI<42 Virus"
+	agent = "H0NI<42.B4n4 Virus"
 	viable_mobtypes = list(/mob/living/carbon/human)
 	permeability_mod = 0.75
-	desc = "If left untreated the subject will probably drive others to insanity."
-	severity = MEDIUM
+	desc = "If left untreated the subject will probably drive others to insanity and go insane themselves."
+	severity = MINOR
 
 /datum/disease/pierrot_throat/stage_act()
 	if(!..())
 		return FALSE
+
+	var/mob/living/carbon/human/H = affected_mob
+
 	switch(stage)
-		if(1)
-			if(prob(10))
-				to_chat(affected_mob, "<span class='danger'>You feel a little silly.</span>")
 		if(2)
-			if(prob(10))
-				to_chat(affected_mob, "<span class='danger'>You start seeing rainbows.</span>")
+			if(prob(4))
+				to_chat(H, "<span class='danger'>You feel [pick("a little silly", "like making a joke", "in the mood for giggling", "like the world is a little more vibrant")].</span>")
+			if(prob(4))
+				to_chat(H, "<span class='danger'>You see [pick("rainbows", "puppies", "banana pies")] for a moment.</span>")
 		if(3)
-			if(prob(10))
-				to_chat(affected_mob, "<span class='danger'>Your thoughts are interrupted by a loud <b>HONK!</b></span>")
+			if(prob(2))
```

Comment:
There's a bit of code duplication here, do you think you could try to organize this a little better?